src/actions/GridActions.js

```diff
@@ -238,7 +238,11 @@ export const doRemoteSort = ({
         );
 
         if (typeof dataSource === 'function') {
-            return dataSource({pageIndex}, {}, sortParams).then((response) => {
+            return dataSource(
+                {pageIndex},
+                filterFields,
+                sortParams
```

Comment:
This becomes a breaking change if filterFields is ordered before sortParams, whereas it's probably just a minor change if it comes second. Do you think it's OK if we keep the order as it was before, appending the new arg?